packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -12,38 +11,22 @@ import { LanguageSelection } from './languageSelection';
 import { trackUserImpression } from './libs/eventTracking';
 import { CodeByteEditorProps } from './types';
 
-const editorStates = states({
-  isIFrame: { height: '100vh' },
```

Comment:
@BandanaKM we should be able to override height and width in portal app by just passing in the props

Reply:
(side note: we want width 100% in monolith too, where did 43 rem come from? just for the styleguide preview?)

Reply:
yeah, the `43rem` was probably an oversight! i was just trying to get a default sizing that resembled what we had on forums/the homepage so that storybook would look visually the same.

Reply:
also totally fine with removing the state utility, since it seems like we no longer need it!